README.md

```diff
@@ -66,6 +66,7 @@ Note: Some of these models seem to have a newer firmware that does not allow loc
 - AC0850/20 AWS_Philips_AIR
 - AC0850/20 AWS_Philips_AIR_Combo
 - AC0850/31
+- AC0850/81 AWS_Philips_AIR_Combo
```

Comment:
Is the firmware part of the model name needed? Normally, this is only needed if there is another AC0850/81 with a different firmware. I suggest to eliminate this and just use AC0850/81 to recognize this model until someone complains.

Reply:
Sure, removed it.